docs/developer-guide/etendo-classic/how-to-guides/how-to-customize-the-etendo-skin.md

```diff
@@ -0,0 +1,118 @@
+---
+title: How to Customize the Etendo Skin
+---
+
+## Overview
+
+The `cssCompile` task in the **Etendo** Gradle configuration is specifically designed to convert `.scss` files into `.css` files. This documentation aims to provide **Etendo partners** with a comprehensive understanding of its function, usage, and benefits.
+
+## Purpose
+
+With modern web development, **SCSS** offers more power and flexibility compared to traditional **CSS**. As Etendo aims to deliver an impeccable user interface, using `.scss` files allows for better organization, variable definitions, and nested rules. However, browsers understand only CSS. Hence, there's a need to **compile SCSS into CSS**, and that's where our `cssCompile` task steps in.
+
+> **Requirements**
+> - npm (Node Package Manager) installed and Sass compiler installed on your system.
+
+!!! info
+    ### Installing npm (Node Package Manager)
+    If you don't have npm installed on your system, follow these steps:
+    
+    - Install npm globally using the following command:
+    
+    ```bash
+    npm install -g npm
```

Comment:
On mac os and windows it is installed with the same commands?
If they are different commands, you can make a tab for each OS.
E.g:
![image](https://github.com/etendosoftware/docs/assets/106354215/e5b22b9e-c0d2-4279-a949-fbd0e4cd8452)

Reply:
Yes Isa, in this case the provided commands are consistent across all platforms (Windows, macOS, and Linux).